website/util/sanitize.py

```diff
@@ -78,7 +78,7 @@ def _ensure_clean(value):
 
 
 # TODO: Remove unescape_entities when mako html safe comes in
-def unescape_entities(value):
+def unescape_entities(value, safe=None):
```

Comment:
@abought, I can confirm that the unescaped file name here will never reach the user / front end. Only the escaped versions of file names are displayed in the registration form and in the admin module. I will definitely add a description of this parameter with a warning to use with caution though, thanks for the suggestion!